### PR TITLE
doc(java): Global tags/extra with logback appender encoder

### PR DESCRIPTION
**Pull request by jeacott1**
adds documentation for global tags and extras using the logback appender.

I'm not sure what the target version is likely to be however? unlikely to remain 1.7.27

### CODE REVIEW

File: src/collections/_documentation/clients/java/integrations.md [hunk 1]
```diff
@@ -811,6 +811,9 @@ Example configuration using the `logback.xml` format:
         <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
             <level>WARN</level>
         </filter>
+        <encoder>
```
Comment (bruno-garcia) [inline src/collections/_documentation/clients/java/integrations.md]:
```suggestion
        <!-- Optionally add an encoder -->
        <encoder>
```

File: src/collections/_documentation/clients/java/integrations.md [hunk 2]
```diff
@@ -845,6 +848,29 @@ void logWithExtras() {
 }
 ```
 
+#### Global Tags
+
+Sometimes its useful to add tags and extra data to all log events.
```
Comment (bruno-garcia) [inline src/collections/_documentation/clients/java/integrations.md]:
```suggestion
Sometimes it's useful to add tags and extra data to all log events.  
```

File: src/collections/_documentation/clients/java/integrations.md [hunk 3]
```diff
@@ -845,6 +848,29 @@ void logWithExtras() {
...
+Tags are distinguished by the existing mdcTags configuration property detailed above.
+
+```java
+  LoggerContext context = (LoggerContext)LoggerFactory.getILoggerFactory();
+  context.putProperty("global", "value");
+```
+
+Global log entries can also be added via third party encoders  
+(*whether such entries can be distinguished as tags or entries however is encoder implementation specific*).
+The net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder for example has a customFields option:
```
Comment (bruno-garcia) [inline src/collections/_documentation/clients/java/integrations.md]:
```suggestion
The `net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder` for example has a `customFields` option:
```

**Comment by bruno-garcia** [comment]
@jeacott1 Thanks for this btw, awesome work!

File: src/collections/_documentation/clients/java/integrations.md [hunk 4]
```diff
@@ -845,6 +849,29 @@ void logWithExtras() {
 }
 ```
 
+#### Global Tags
+
+Sometimes it's useful to add tags and extra data to all log events.  
+Tags and extras can be added to logs globally (not threadbound) by adding entries to the LoggerContext.
```
Comment (MimiDumpling) [inline src/collections/_documentation/clients/java/integrations.md]:
```suggestion
You can add tags and extras to logs globally (not thread-bound) by adding entries to the LoggerContext.  
```

File: src/collections/_documentation/clients/java/integrations.md [hunk 5]
```diff
@@ -845,6 +849,29 @@ void logWithExtras() {
...
+Tags and extras can be added to logs globally (not threadbound) by adding entries to the LoggerContext.  
+Tags are distinguished by the existing mdcTags configuration property detailed above.
+
+```java
+  LoggerContext context = (LoggerContext)LoggerFactory.getILoggerFactory();
+  context.putProperty("global", "value");
+```
+
+Global log entries can also be added via third-party encoders  
+(*whether such entries can be distinguished as tags or entries however is encoder implementation specific*).
```
Comment (MimiDumpling) [inline src/collections/_documentation/clients/java/integrations.md]:
```suggestion
(*whether such entries can be distinguished as tags or entries, however, is encoder implementation-specific*).
```

**Pull request merged**